extensions/notebook/package.nls.json

```diff
@@ -6,6 +6,7 @@
 	"notebook.useExistingPython.description": "Local path to a preexisting python installation used by Notebooks.",
 	"notebook.overrideEditorTheming.description": "Override editor default settings in the Notebook editor. Settings include background color, current line color and border",
 	"notebook.maxTableRows.description": "Maximum number of rows returned per table in the Notebook editor",
+	"notebook.trustedBooks.description": "Books to trust automatically.",
```

Comment:
Hm, I'd consider trying to make it clearer what this is doing. Something that includes mentioning that it will result in child notebooks being trusted (the book itself doesn't really have a concept of trust)